Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 apache-airflow = "*"
 sagetasks = "*"
+boto3 = "*"
```

Comment:
Do you also need to install the AWS provider package?

Reply:
the aws provider package comes with `apache-airflow`, it shows up in `Pipfile.lock `and `requirements.txt`, but not in `Pipfile`

Reply:
I should have just checked the `Pipfile.lock` file. My bad!